src/wolfio.c

```diff
@@ -169,15 +169,17 @@ int BioReceive(WOLFSSL* ssl, char* buf, int sz, void* ctx)
 
     recvd = wolfSSL_BIO_read(ssl->biord, buf, sz);
     if (recvd <= 0) {
-        if (wolfSSL_BIO_supports_pending(ssl->biord) &&
+        if (/* ssl->biowr->wrIdx is checked for Bind9 */
+            ssl->biowr != NULL && ssl->biowr->type == WOLFSSL_BIO_BIO &&
+            ssl->biowr->wrIdx != 0 &&
```

Comment:
Is this a check for pending write?
I believe the OpenSSL API is BIO_wpending(). So add wolfSSL_BIO_wpending().